Docker/php/remp.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+BASE=$PWD
```

Comment:
Is this actually necessary? When the script is run, it is expected to be run from the root of the repository and then goes to `cd ${APP_NAME}`.

So to link the package, it should be enough to call this, isn't it?

```
cd Package/remp
yarn link
cd -
```

Reply:
That wouldn't work as the working dir at that stage is already inside ${APP_NAME}
eg. `/var/www/html/Beam`

But I look for all options of `yarn link --help` and found how to specify folder from which to link and where to link also. 
Pushed updated version